src/snowflake/cli/_plugins/streamlit/commands.py

```diff
@@ -204,10 +204,15 @@ def _migrate_v1_streamlit_to_v2(pd: ProjectDefinition):
     if pd.streamlit.database:
         identifier["database"] = pd.streamlit.database
 
+    if pd.streamlit.name.startswith("<!") and pd.streamlit.name.endswith("!>"):
```

Comment:
Same here, why we use `snow init` templates?

Reply:
My bad, it was meant to be `"<%"`